php/www/auth/index.php

```diff
@@ -0,0 +1,60 @@
+<?php
+session_start();
+if (isset($_SESSION['user_id'])) {
+    // セッションがない場合はサインインへリダイレクト
+    header("Location: /");
+    exit;
+}
+?>
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>学内掲示板 - サインイン</title>
+    <?php require_once('../lib/bootstrap.php'); ?>
+</head>
+
+<body>
+    <div class="container mt-5 w-50">
```

Comment:
`m-5`で上部から固定値で中央に寄せようとしない、`w-50`だとモバイル時に崩れるので消す。
`container`で`max-width`がついてるのでそれで十分。

```suggestion
    <div class="container  h-100 d-flex justify-content-center align-items-center">
```